src/CalcManager/CalculatorVector.h

```diff
@@ -141,7 +135,7 @@ class CalculatorVector
             }
 
             std::wstring expressionSuffix{};
-            hr = GetExpressionSuffix(&expressionSuffix);
+            hr = GetExpressionSuffix(&expressionSuffix); // Will always return S_OK
```

Comment:
I would suggest to not add this comment, there is a risk this comment will become obsolete if the implementation of `GetExpressionSuffix` changes and will be misleading for developers.